### ISSUE TITLE
Voting on videos?

### ISSUE THREAD

**Issue by lrq3000** [lbryio/lbry-desktop#3429]
I have read the roadmap and any developers update and documentation I could find, but I found no mention about a future implementation of voting on videos. I saw #2598 but it discusses voting on comments for moderation purposes, which is different.

I think voting is an important feature to have for two reasons:

1. because it allows the users to signal what content they like (or dislike) which is [very useful metric for the content creator](https://www.youtube.com/watch?v=fdGbOT8NXnE).

2. this can help with the recommender system to triage what videos to recommend (in addition to the views count for example).

3. users are simply too used to voting by now, and [some may pass on trying LBRY because of this lack](https://www.reddit.com/r/dtube/comments/e7kwyu/how_do_i_earn_on_dtube/fckf1fs?utm_source=share&utm_medium=web2x).

Is voting on videos planned at some point?

**Comment by tzarebczan** [comment]
Thanks for the feedback. Voting on LBRY is currently done through the tip and supports mechanism (lbry.com/faq/tipping / lbry.com/faq/trending) but we don't expose some of the values in the app yet. I don't think we'll have a different voting mechanism, and this one will continue to evolve UI wise.

**Comment by lrq3000** [comment]
Hello, thank you for the reply. I have experimented a bit more with lbry, and I think I understand how it's intended to work. I noticed that there were LBC rewards everyday just for watching content, so if this continues indefinitely (and is not just a temporary giveaway until LBRY is more widespread), indeed this mechanism gives the possibility to any viewer to cast a certain number of votes/tips per day.

However, it still is not like a voting system, particularly because there is no dislike. So I still would strongly suggest to reconsider the possibility to add such a voting system. Maybe not in the blockchain, and there is no need to monetize votes since there is the tip system in place, but votes just for themselves would have great value to gauge content quality and feed the search engine and recommender system.
